src/main/java/org/eclipse/xpanse/terraform/boot/models/enums/DestroyScenario.java

```diff
@@ -11,37 +11,35 @@
 import org.eclipse.xpanse.terraform.boot.models.exceptions.UnsupportedEnumValueException;
 
 /**
- * The deployment scenario is used to set into TerraformResult and sent back to the
- * client which sent the deployment request in what scenario.
+ * Destroy Scenario. The destroy scenario is used to set into TerraformResult and sent back to the
+ * client which sent the destroy request in what scenario.
  */
-public enum DeploymentScenario {
-    DEPLOY("deploy"),
-    MODIFY("modify"),
+public enum DestroyScenario {
```

Comment:
@baixinsui My idea/requirement is to completely remove this enum itself. 
And use a different webhook in xpanse for rollback, purge and destroy. and then when the update comes via the specific webhook, we know which usecase it is for and handle it accordingly.

Reply:
This means we need add rollbackService, destroyService and purgeService. Only in this way can we obtain the destruction scenario within the Terraform boot